app/routes/$repo.tsx

```diff
@@ -42,12 +42,14 @@ export default function Repo() {
   const { content } = useLoaderData<LoaderData>()
   const matches = useMatches()
   const [match] = matches.slice(-1)
+  const path = match.params["*"] || "index"
 
   return (
     <InternalPage
-      activePath={match.params["*"] || "index"}
+      activePath={path}
       contentDisplayName={content.displayName}
       contentPath={content.contentName}
+      header={path === "index" ? content.landingHeader : undefined}
```

Comment:
can you please merge in what's on `main` and see if this works when you go to `http://localhost:3000/cadence` ? might need to handle a null/blank/undefined path